test/verify/check-machines

```diff
@@ -195,9 +195,6 @@ class TestMachines(MachineCase):
         b.wait_present("#vm-subVmTest1-state")
         b.wait_in_text("#vm-subVmTest1-state", "running")
 
-        xml = DOMAIN_XML.format(**args)
-        m.execute("echo \"{0}\" > /tmp/xml && virsh define /tmp/xml && virsh start {1}".format(xml, name))
-
         m.execute('[ "$(virsh domstate {0})" = running ] || '
                   '{{ virsh dominfo {0} >&2; cat /var/log/libvirt/qemu/{0}.log >&2; exit 1; }}'.format(name))
```

Comment:
That is redundant with `startVm()` as well. But it doesn't hurt really, and this is crucial to get in.